scripts/common.py

```diff
@@ -120,6 +121,13 @@
     OPPIA_TOOLS_DIR, 'redis-cli-%s' % REDIS_CLI_VERSION,
     'src', 'redis-cli')
 
+ES_PATH = os.path.join(
+    OPPIA_TOOLS_DIR, 'elasticsearch-%s' % ELASTICSEARCH_VERSION)
+ES_PATH_CONF = os.path.join(
```

Comment:
ES_PATH_CONFIG_FILE or ES_PATH_CONFIG_DIR (not sure which, but let's make it clear in the name)

Reply:
Done